examples/go/tagging/.vcconfig

```diff
@@ -0,0 +1,8 @@
+[vcenter]
```

Comment:
Why did you make that a hidden file instead of the `vcconfig.toml` we use across the examples?

Reply:
Good point--the hidden file format will not work. I'll make it vcconfig.toml.